packages/rpc-core/src/rpc-methods/getFeeForMessage.ts

```diff
@@ -0,0 +1,18 @@
+import { Base64EncodedBytes, Commitment, RpcResponse, Slot, U64UnsafeBeyond2Pow53Minus1 } from './common';
+
+/** Fee corresponding to the message at the specified blockhash */
+type GetFeeForMessageApiResponse = RpcResponse<U64UnsafeBeyond2Pow53Minus1 | null>;
```

Comment:
The RPC does not wrap this in a `RpcResponse` wrapper.

```diff
- type GetFeeForMessageApiResponse = RpcResponse<U64UnsafeBeyond2Pow53Minus1 | null>;
+ type GetFeeForMessageApiResponse = U64UnsafeBeyond2Pow53Minus1 | null;
```

Reply:
I believe it does mate

https://docs.solana.com/api/http#getfeeformessage
https://github.com/solana-labs/solana/blob/67d6d688ccf10d09a012559a43f524aa35e87919/rpc/src/rpc.rs#L3996
https://github.com/solana-labs/solana/blob/67d6d688ccf10d09a012559a43f524aa35e87919/rpc/src/rpc.rs#L120

Reply:
Oh, oops. I was reading just the docs, which are wrong. They should mention that an RpcResponse object is returned.

Reply:
Yeah looks like it's in their payload just not the description like the others